src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -28,6 +28,8 @@ public class ModelManager implements Model {
     private final FilteredList<Job> filteredJobs;
     private final FilteredList<Company> filteredCompanies;
 
+    private Company highlightedCompany;
```

Comment:
Could there be a way to get this to be private final?

Reply:
I would need to have the constructor for ModelManager to include a highlightedCompany field